Dockerfile.openshift

```diff
@@ -0,0 +1,13 @@
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
```

Comment:
You can do `Dockerfile` and `Dockerfile.rhel`, unless there is a docker file in the repo already that you don't want to overwrite.

Reply:
There's already a `Dockerfile`